server/apps/distributed-app/docker-compose-crowdsec.yml

```diff
@@ -0,0 +1,113 @@
+version: '3'
+
+services:
+
+  james:
+    depends_on:
+      cassandra:
+        condition: service_healthy
+      opensearch:
+        condition: service_started
+      rabbitmq:
+        condition: service_started
+      s3:
+        condition: service_started
+      pulsar:
+        condition: service_started
+    image: apache/james:distributed-latest
+    container_name: james
+    hostname: james.local
+    command:
+      - --generate-keystore
+    networks:
+      - james
+    ports:
+      - "80:80"
+      - "25:25"
+      - "110:110"
+      - "143:143"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+      - "8000:8000"
+    volumes:
+      - ./sample-configuration/pulsar.properties:/root/conf/pulsar.properties
+      - ./sample-configuration/queue-pulsar.properties:/root/conf/queue.properties
+      - ./logs:/logs/
```

Comment:
Create a docker volume that is shared between James container and CrowdSec, rather than volume to the host file system?

Reply:
Make sense! I will fix it